src/collections/_documentation/learn/configuration.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Configuration
+sidebar_order: 1
+---
+
+SDKs are configurable in many ways.  The options are largely standardized between SDKs but there are
+some differences to better acomodate platform peculiarities.  Options are set when the SDK is first
+initialized.
+
+{% include components/platform_content.html content_dir='config-intro' %}
+
+## Common Options
+
+The list of common options across SDKs.  These work more or less the same in all SDKs but some
+subtle differences will exist to better support the platform.
+
+{:.config-key}
+### debug
+
+Turns debug mode on or off.  If debug is enabled SKD will attempt to print out useful debugging
+information if something goes wrong with sending the event.  The default is always `false` and
+it's generally not recommended to turn it on in production but doing so will not cause any
+safety concerns.
+
+{:.config-key}
+### release
+
+Sets the release.  Some SDKs will try to automatically configure a release out of the box but
+if you have the chance it's a better idea to manually set it.  That way it's guaranteed to be
+in sync with your deploy integrations or sourcemap uploads.
+
+Release names are just strings but some formats are detected by Sentry and might be rendered
+differently.  For more information have a look at [the releases documentation]({% link _documentation/learn/releases.md %}).
+
+{:.config-key}
+### environment
+
+Sets the environment.  This string is freeform and not set by default.  A release can be associated
+with more than one environment to separate them im the UI (think `staging` vs `prod` or similar).
+
+{:.config-key}
+### repos
+
+For repository integration the repository identifiers need to be sent along.  For more information
+have a look at [the releases documentation]({% link _documentation/learn/releases.md %}).
+
+{:.config-key}
+### sample-rate
+
+Configures the sample rate as a percentage of events to be sent in the range of `0.0` to `1.0`.  The
+default is `1.0` which means that 100% of events are sent.  If set to `0.1` only 10% of events will
+be sent.  Events are picked randomly.
+
+{:.config-key}
+### max-breadcrumbs
+
+This variable controls the total amount of breadcrumbs that should be captured.  This defaults
```

Comment:
Not sure if this is documented elsewhere, but this seems like a good place to tell how breadcrumbs work (i.e. it's a scrollback buffer, and this controls the size of that buffer)